using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+  has data, the continuous view will not appear to have data in it until the
+  first scheduled run completes. In general continuous views will be updated
+  automatically on a schedule but you can also do it manually by running
+  `REFRESH MATERIALIZED VIEW <continuous_view_name>` ([API Docs][refresh-continuous-view]).
+
+To select data from the continuous aggregates, simply run a `SELECT` query on
+the continuous view, for instance, to return the average and spread for the
+first quarter of 2018 for device `D132` you would run:
+```sql
+SELECT * FROM device_summary 
+WHERE device_id = 'D132' 
+  AND bucket > '2018-01-01' AND bucket < '2018-04-01';
+```
+Or we can do more complex queries on the aggregates themselves, for instance, if
+we wanted to know the top 20 largest metric spreads in that quarter, we could do
+something like: 
+```sql
+SELECT * FROM device_summary 
+WHERE bucket > metric_spread > 1800
+  AND bucket > '2018-01-01' AND bucket < '2018-04-01'
+ORDER BY bucket DESC, device_id DESC LIMIT 20;
+```
+---
+
+### Altering and Dropping Continuous Views [](alter)
+Only the `WITH` option `timescaledb.continuous` is required to be set in order to
+create the continuous view, however there are several other options that may
+also be specified when creating the view, we'll discuss a few important ones in
+the [best practices][best-practices] section and you can find the full list in
+the [API docs][create-continuous-agg]. They may be set in the `WITH` options of
+the initial create view, or they may be set by running an `ALTER VIEW` command
+like so: 
+```sql
+ALTER VIEW device_summary SET (timescaledb.refresh_interval = '10 min');
+```
+Which sets the interval at which the view refreshes to 10 minutes.
+
+Other alterations to the continuous aggregate view are currently disallowed, to
+alter a continuous aggregate view in other ways it must be dropped and
+re-created, this can entail some time to re-calculate aggregations. 
+
+A continuous aggregate may be dropped by using the `DROP VIEW` command. 
+```sql
+DROP VIEW device_summary;
+```
+
+---
+
+### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
+When dropping data in a raw hypertable using the [`drop_chunks`
+function][api-drop-chunks] that has a continuous aggregate created on it, you
+must specify the `cascade_to_materializations` argument to the `drop_chunks`
+call. Currently, the only option for this argument is `true`, which will cause
+the materialization to drop all data associated with any chunks dropped from the
+raw hypertable. Further data retention options are planned for future releases
+(see [future work](#future-work). 
+
+The same argument must also be supplied to the [`add_drop_chunks_policy`
+function][api-add-drop-chunks] when creating a data retention policy for a
+hypertable with a continuous aggregate.  
+
+---
+
+### Best Practices [](best-practices)
+
+**Using `timescaledb.information` Views** 
```

Comment:
this feels like it should be it's own section and not in best-practices. We should also talk about and explain the thresholds here.